extide/gradle/src/org/netbeans/modules/gradle/api/GradleBaseProjectBuilder.java

```diff
@@ -242,6 +244,23 @@ void processDependencies() {
         problems.addAll(unresolvedProblems.values());
     }
 
+    private ModuleDependency resolveModuleDependency(File gradleUserHome, String c) {
+        GradleModuleFileCache21 moduleCache = GradleModuleFileCache21.getGradleFileCache(gradleUserHome.toPath());
+        GradleModuleFileCache21.CachedArtifactVersion artVersion = moduleCache.resolveModule(c);
+        Set<File> binaries = artifactSore.getBinaries(c);
```

Comment:
This makes the module dependency resolution more stable. If the ArtifactStore has no information on a module for whatever reason, NetBeans tries to resolve it using the Gradle Module Cache directory structure.